be/src/formats/json/numeric_column.h

```diff
@@ -0,0 +1,19 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+#pragma once
+
+#include <limits>
+#include <string>
+
+#include "column/column.h"
+#include "common/status.h"
+#include "runtime/types.h"
+#include "simdjson.h"
+
+namespace starrocks::vectorized {
+
+template <typename T>
+Status add_numeric_column(Column* column, const TypeDescriptor& type_desc, const std::string& name,
+                          simdjson::ondemand::value& value);
```

Comment:
If `value` will be modified inside, pass it by pointer, otherwise pass it by const reference.

Reply:
Updated.